tests/Browser/Pages/Frontend/HomePage/HomePageTest.php

```diff
@@ -0,0 +1,264 @@
+                    ->assertTitle('Home page')
+                    ->assertSee('Representative Hotels')
+                    ->clickLink('Hotels')
+                    ->assertTitle('LIST HOTELS')
+                    ->assertSee('List Hotels')
+                    ->assertPathIs('/hotels');
+        }); 
+    }
+
+    /**
+     * Test link list News.
+     *
+     * @return void
+     */
+    public function testLinkNews()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/')
+                    ->assertTitle('Home page')
+                    ->assertSee('Representative Hotels')
+                    ->clickLink('News')
+                    ->assertTitle('LIST NEWS')
+                    ->assertSee('TOP NEWS')
+                    ->assertPathIs('/news');
+        }); 
+    }
+
+    /**
+     * Test link login.
+     *
+     * @return void
+     */
+    public function testLinkLogin()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->logout();
+            $browser->visit('/')
+                    ->assertTitle('Home page')
+                    ->assertSee('Representative Hotels')
+                    ->clickLink('Login')
+                    ->assertTitle('Login')
+                    ->assertSee('Login')
+                    ->assertPathIs('/login');
+        }); 
+    }
+
+    /**
+     * Test link register.
+     *
+     * @return void
+     */
+    public function testLinkRegister()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->logout();
+            $browser->visit('/')
+                    ->assertTitle('Home page')
+                    ->assertSee('Representative Hotels')
+                    ->clickLink('Register')
+                    ->assertTitle('Register')
+                    ->assertSee('Register')
+                    ->assertPathIs('/register');
+        }); 
+    }
+
+    /**
+     * Test link feedback
+     *
+     * @return void 
+     */
+    public function testLinkFeedback()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/')
+                    ->assertTitle('Home page')
+                    ->assertSee('Representative Hotels')
+                    ->clickLink('Feedback')
+                    ->assertTitle('Feedback')
+                    ->assertSee('Feedback')
+                    ->assertPathIs('/sendfeedback/create');
+        });
+    }
+
+    /**
+     * Make data for test.
+     *
+     * @return void
+     */
+    public function makeData($row)
+    {   
+        factory(Place::class, $row)->create();
+        factory(User::class, $row)->create();
+        factory(Guest::class, $row)->create();
+        $placeIds = Place::all('id')->pluck('id')->toArray();
+        $faker = Faker::create();
+        for ($i = 0; $i < $row; $i++) {
+            factory(Hotel::class, 1)->create([
+                'place_id' => $faker->randomElement($placeIds)
```

Comment:
trong này chạy randomElement thì dữ liệu chạy test sao mà chính xác được.
vì mình lấy mảng id ra rồi thì foreach theo nó luôn chứ ko nên random dữ liệu , trong factory truyền vào row là được. 
thay for row bằng foreach theo $placeIds của nhé . phía dưới tương tự vs hotelIds  & roomIds